pallets/dkg-proposal-handler/src/lib.rs

```diff
@@ -116,34 +121,98 @@ pub mod pallet {
 			_origin: OriginFor<T>,
 			prop: ProposalType,
 		) -> DispatchResultWithPostInfo {
+			// let sender = ensure_signed(origin)?;
```

Comment:
Should there be a TODO here? Is there anything wrong with having the origin be any `Signed` origin?

Reply:
Nope, it is good, as we check for the data and signature but we don't want to need the check anyway.
We can uncomment it.

Reply:
We should also check the signature to see if the signer is a dkg authority, that's not been done currently.

Reply:
Yes, that's correct! we should check that against the current dkg public key!

Reply:
If we don't check that it is signed shady then anyone can call this and we can't charge tx fees. That means it can be spammed potentially?

Reply:
@Wizdave97 could you create a task for that? so maybe someone could work on it?